src/components/Lobby/Lobby.tsx

```diff
@@ -11,22 +13,24 @@ interface Props {
 function Lobby({players}: Props) {
     const [sessionGame, setSessionGame] = useState({});
     const [sessionPlayers, setSessionPlayers] = useState([]);
-    console.log('sessionGame', sessionGame)
     const location = useLocation();
     const game = location.state;
     const { gameid } = useParams();
-    const joinURL = `http://localhost:3000/Brain-Defrost_FE/join/${gameid}`;
     const navigate = useNavigate();
+    const [joinURL, setJoinUrl] = useState('');
+    const [error, setError] = useState<string>('');
 
     useEffect(() => {
         // @ts-expect-error
         const sessionGame = JSON.parse(sessionStorage.getItem('game'))
         setSessionGame(sessionGame)
+        let stringQuestion = JSON.stringify(game)
```

Comment:
This was a cool way to solve that issue! Could these statements be defined in a separate function and then invoked here?